com.airtame.Client.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.airtame.Client</id>
+  <name>Airtame</name>
+  <summary>Airtame is a wireless streaming solution</summary>
+  <developer_name>Airtame</developer_name>
+  <description>
+  <p>Airtame is a small wireless HDMI device that plugs into the HDMI port of any screen or projector. Unlike other wireless HDMI devices, nothing is plugged into your computer or smartphone. Simply download our app and stream your content to the screen from any major computer platform, tablet, or smartphone.</p>
+</description>
```

Comment:
Can you add a note that makes it clear this isn't offered directly by Airtame themselves like this? https://github.com/flathub/com.slack.Slack/blob/master/com.slack.Slack.appdata.xml#L17

Reply:
> Can you add a note that makes it clear this isn't offered directly by Airtame themselves like this? https://github.com/flathub/com.slack.Slack/blob/master/com.slack.Slack.appdata.xml#L17

Done